mysql/tests/compose/percona.yaml

```diff
@@ -0,0 +1,56 @@
+services:
+  mysql-master:
```

Comment:
Nit, do we want to swap these to `mysql-primary` / `mysql-replica`? I see this is already how the are in the mysql/mariadb compose files. Can also clean this up in a follow up